packages/admin/cms-admin/src/dam/FileForm/FileSettingsFields.tsx

```diff
@@ -115,6 +115,7 @@ export const FileSettingsFields = ({ isImage, folderId }: SettingsFormProps): Re
                         name="license.type"
                         label={<FormattedMessage id="comet.dam.file.type" defaultMessage="Type" />}
                         fullWidth
+                        required={damConfig.requireLicenseFeature}
```

Comment:
Currently, there is no clear indicator telling the user that the license fields are suddenly required (except for the *). If I change the alt text, the save button stays disabled. This might be confusing. 

Also you should validate that the license type isn't set to NO_LICENSE. Because the user can fill out all fields, then set the type to NO_LICENSE and submit. Then, all other license data is omitted in the API. 
If you set `shouldShowError={() => true}` for the field, the error is always shown. Then you don't need an additional hint.

Reply:
I`ve added your suggestions and uploaded a new video. In the end of the video, when switching from ROYALTY_FREE to another type, the `durationTo` field will only be validated again when editing some content. Do you know how to change that? I already tried `validateOnBlur` on the form itself.

Reply:
Oh that's tricky. You can add `validateFields={["license.durationTo"]}` to the license.type field and `validateOnBlur` to the form.

Sadly, it doesn't work without `validateOnBlur` because the durationTo field has to be rerendered before the validation. So I don't see another way.